powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -196,21 +206,26 @@ public class PowerAuthServiceConfiguration {
      * HTTP connection timeout.
      */
     @Value("${powerauth.service.http.connection.timeout}")
-    private Integer httpConnectionTimeout;
+    private Duration httpConnectionTimeout;
 
     /**
-     * Token timestamp validity in milliseconds, checked before validating the token.
+     * Specifies the validity duration of token timestamps, checked before token validation.
+     * <p>
+     * The value is specified in the ISO-8601 duration format PTnHnMn.nS.
      */
     @Value("${powerauth.service.token.timestamp.validity}")
-    @Min(1)
-    private long tokenTimestampValidityInMilliseconds;
+    @DurationMin(millis = 1)
+    private Duration tokenTimestampValidity;
 
     /**
-     * Token timestamp validity to future in milliseconds, checked before validating the token.
+     * Specifies the validity duration to future of token timestamps, checked before token validation.
+     * <p>
+     * The value is specified in the ISO-8601 duration format PTnHnMn.nS.
      */
     @Value("${powerauth.service.token.timestamp.forward.validity}")
-    @Min(0)
-    private long tokenTimestampForwardValidityInMilliseconds;
+    @DurationMin(millis = 0)
```

Comment:
What is the difference between this and the previous format `@Min(0)`?

Reply:
The @Min apparently wont work on type Duration